packages/dx-react-grid/docs/reference/virtual-table.md

```diff
@@ -1,28 +1,68 @@
+layoutComponent | ElementType&lt;[TableLayoutProps](table.md#tablelayoutprops)&gt; | | A component that renders a table layout.
+cellComponent | ElementType&lt;[TableDataCellProps](table.md#tabledatacellprops)&gt; | | A component that renders a table cell.
+rowComponent | ElementType&lt;[TableDataRowProps](table.md#tabledatarowprops)&gt; | | A component that renders a table row.
+noDataCellComponent | ElementType&lt;[TableNoDataCellProps](table.md#tablenodatacellprops)&gt; | | A component that renders a table cell when the table is empty.
+noDataRowComponent | ElementType&lt;[TableRowProps](table.md#tablerowprops)&gt; | | A component that renders a table row when the table is empty.
+stubCellComponent | ElementType&lt;[TableCellProps](table.md#tablecellprops)&gt; | | A component that renders a stub table cell if the cell value is not provided.
+stubHeaderCellComponent | ElementType&lt;[TableCellProps](table.md#tablecellprops)&gt; | | A component that renders a stub header cell if the cell value is not provided.
+messages | object | | An object that specifies the [localization messages](#localization-messages).
 
-This plugin also supports the [Table](table.md#properties) plugin's properties.
+## Localization Messages
+
+An object with the following shape:
+
+Field | Type | Default | Description
+------|------|---------|------------
+noData? | string | 'No data' | Specifies text that is displayed when the Grid does not contain data.
+
+## Plugin Components
+
+Name | Properties | Description
+-----|------------|------------
+VirtualTable.Cell | [TableDataCellProps](table.md#tabledatacellprops) | Renders a table data cell.
+VirtualTable.Row | [TableDataRowProps](table.md#tabledatarowprops) | Renders a table data row.
+VirtualTable.NoDataCell | [TableCellProps](table.md#tablecellprops) | Renders a table cell when the table is empty.
+VirtualTable.NoDataRow | [TableRowProps](table.md#tablerowprops) | Renders a table row when the table is empty.
+VirtualTable.StubCell | [TableCellProps](table.md#tablecellprops) | Renders a stub table cell.
+VirtualTable.StubHeaderCell | [TableCellProps](table.md#tablecellprops) | Renders a stub table header cell.
+
+If you specify additional properties, they are added to the component's root element.
 
 ## Plugin Developer Reference
 
 ### Imports
 
-Same as the [Table](table.md#imports)'s.
+Name | Plugin | Type | Description
+-----|--------|------|------------
+rows | Getter | Array&lt;any&gt; | Rows to be rendered by the virtual table view.
+columns | Getter | Array&lt;[Column](table.md#column-extension)&gt; | Columns the virtual table view should render.
+getRowId | Getter | (row: any) => number &#124; string | A function used to get a unique row identifier.
+getCellValue | Getter | (row: any, columnName: string) => any | A function used to get the column value for a given row.
```

Comment:
getCellValue -> gets the column value for a row? The name and description don't to match.